dss_metrics/minio_rest_collector.py

```diff
@@ -44,7 +44,10 @@ def __init__(self, configs, whitelist_patterns, filter=False):
         self.configs = configs
         self.whitelist_patterns = whitelist_patterns
         self.filter = filter
-        self.minio_metrics = {'minio_disk_storage_used_bytes'}
+        self.minio_metrics = {'minio_disk_storage_used_bytes',
+                              'minio_disk_storage_total_capacity_bytes'}
+        self.cluster_id_url_prefix = "http://"
```

Comment:
why we need multiple url_prefix when there is no change in the text. You can club line 49 and 51 together